templates/bake/element/Controller/index.twig

```diff
@@ -26,6 +26,9 @@
             ->contain({{ Bake.exportArray(belongsTo)|raw }});
 {% else %}
         $query = $this->{{ currentModelName }}->find();
+{% endif %}
+{% if Bake.hasPlugin('Authorization') %}
+        $query = $this->Authorization->applyScope($query);
```

Comment:
Do we need to bake stub code for these?

Reply:
I think it is helpful. I usually use this to build in permissions like 'view all the projects I have membership in'.

Reply:
I mean, should we bake/generate the authorization code stubs to match this?

Reply:
> I mean, should we bake/generate the authorization code stubs to match this?

That can be done already with `bake policy --type table Articles`. If we wanted to get fancy we could make the bake code generation respect whether or not you have a table/entity policy generated.